### ISSUE TITLE
Resurrect "Order revisions by date" option.

### ISSUE THREAD

**Issue by Thuffir** [gitextensions/gitextensions#6412]
## Feature description
In **2.x.x** I was able to switch between commit sorting by **topology** or by **date** (`View -> Order revisions by date`). I think it is a very important option for cherry picking and for branch comparison.

Unfortunately this option is gone in **3.x.x** (#6411). I think it would be worth to resurrect it.

## Environment
- Git Extensions 3.0.2.5232
- Build 23b6f51905006ccdde8701591df706284b4155dc
- Git 2.20.1.windows.1
- Microsoft Windows NT 10.0.17763.0
- .NET Framework 4.7.3324.0
- DPI 96dpi (no scaling)

**Comment by RussKie** [comment]
We won't be doing this as it would involve re-writing the graph. This is neither quick nor easy.

**Comment by Thuffir** [comment]
OK Thanks for the information. One more reason to stay with 2.51.05.

**Comment by pmiossec** [comment]
>One more reason to stay with 2.51.05.

I'm curious to know the others... (as I see a lot of improvements over the v2).

Or this is just an offensive remark over contributors that are passing a lot of time in trying to improve this good tool. We can't please everyone and some difficult choices should be made. This one was one, removing linux support was another.

**Comment by Thuffir** [comment]
Sorry, I did not meant to be offensive. By no means. Probably should have written "One more reason to stay with 2.51.05 for the moment." The other reason is that I am being hit by huge performance issues (probably #5884 or #6357). And at the moment 2.51.05 just works better for me. When those issues get fixed I will probably work around by starting gitk when I need the topology sorting.

It is a great tool. Actually the best GIT front-end I have ever used. This is not the reason I am switching back to 2.51 but the performance penalty which surely will be fixed soon. So keep up the good work and sorry for confusion.

**Comment by kasal** [comment]
Mee too!
I mean, the absence of toposort is a showstopper for me.

> I'm curious to know the others... (as I see a lot of improvements over the v2).

Well, I noticed one another annoyance in 3.x: in the diff panel, the current file name is not highlighted.


But I do not have to return to 2.x branch: I resurrected an early devel version, labeled 3.00.00.2788Dev@82d8f.  It has topo order, working diff panel, yet it has some of the most important improvements (branches displayed in a separate panel, as a tree, has command to delete remote branches).

Lucky that the program is so good and stable that I can use even early alpha build for regular work.
And I agree, this is really the best git GUI, I enjoy that!  (It's just that the latest version is not the best one, but any version is far better than any other graphical front end.)
